letsencrypt/client/client.py

```diff
@@ -23,6 +28,28 @@
 from letsencrypt.client.display import enhancements
 
 
+SETUPTOOLS_AUTHENTICATORS_ENTRY_POINT = "letsencrypt.authenticators"
+"""Setuptools entry point group name for Authenticator plugins."""
+
+
+def init_auths(config):
```

Comment:
This function seems more like a main.py or scripts function rather than belonging in Client. Though I also see the value in keeping as much complexity out of main.py as possible.  I feel that we may soon need another init type file.  Thoughts?